### ISSUE TITLE
can't reproduce the SwAV STL-10 example

### ISSUE THREAD

**Issue by chtlp** [Lightning-AI/lightning-bolts#810]
## 🐛 Bug

Hi, I can't reproduce the SwAV STL-10 model results using instructions from the document:
https://pytorch-lightning-bolts.readthedocs.io/en/latest/self_supervised_models.html#id21

### To Reproduce

Because I don't have a V100, I am training with 1 or 2 2080ti. I restricted the batch size to 64 because of the smaller GPU memory.

Training with 2 GPUs:
```
python -m vault_embed.swav.swav_module \
    --online_ft \
    --gpus 2 \
    --optimizer lars \
    --batch_size 64 \
    --learning_rate 1e-3 \
    --gaussian_blur \
    --queue_length 0 \
    --jitter_strength 1. \
    --nmb_prototypes 512 \
    --dataset stl10 --data_dir /run/shm/stl-10 \
    --fast_dev_run 0
```

Training with 1 GPU:
```
python -m vault_embed.swav.swav_module \
    --online_ft \
    --gpus 1 \
    --optimizer lars \
    --batch_size 64 \
    --learning_rate 1e-3 \
    --gaussian_blur \
    --queue_length 0 \
    --jitter_strength 1. \
    --nmb_prototypes 512 \
    --dataset stl10 --data_dir /run/shm/stl-10 \
    --fast_dev_run 0
```

The training all stuck at a high training loss without making further progress

```
Epoch 44:  99% 778/783 [27:11<00:10,  2.10s/it, loss=6.24, v_num=13]
Epoch 44:  99% 779/783 [27:11<00:08,  2.09s/it, loss=6.24, v_num=13]
Epoch 44: 100% 780/783 [27:12<00:06,  2.09s/it, loss=6.24, v_num=13]
Epoch 44: 100% 781/783 [27:12<00:04,  2.09s/it, loss=6.24, v_num=13]
```

![Xnip2021-05-03_18-12-52](https://user-images.githubusercontent.com/568535/116864912-352b2500-ac3b-11eb-9a8d-1fbe1f73ab53.jpg)

Really important for my research to get this model working, let me know if more information should be provided. Many thanks!!

### Environment

```
python collect_env_details.py
```

* CUDA:
	- GPU:
		- GeForce RTX 2080 Ti
		- GeForce RTX 2080 Ti
		- GeForce RTX 2080 Ti
		- GeForce RTX 2080 Ti
		- GeForce RTX 2080 Ti
		- GeForce RTX 2080 Ti
	- available:         True
	- version:           10.2
* Packages:
	- numpy:             1.20.0
	- pyTorch_debug:     False
	- pyTorch_version:   1.8.1+cu102
	- pytorch-lightning: 1.2.10
	- tqdm:              4.60.0
* System:
	- OS:                Linux
	- architecture:
		- 64bit
		- ELF
	- processor:         x86_64
	- python:            3.9.1
	- version:           #202011181948 SMP Wed Nov 18 20:49:19 UTC 2020
